modular_skyrat/master_files/code/modules/clothing/back/antigravityharness.dm

```diff
@@ -0,0 +1,283 @@
+#define MODE_GRAVOFF "Off"
+#define MODE_ANTIGRAVITY "Anti-Gravity Field"
+#define MODE_EXTRAGRAVITY "Extra-Gravity Field"
+#define GRAVITY_FIELD_COST 20
+#define OFF_STATE "gravityharness-off"
+#define ANTIGRAVITY_STATE "gravityharness-anti"
+#define EXTRAGRAVITY_STATE "gravityharness-extra"
+
+/obj/item/gravityharness
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/backpacks.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/back.dmi'
+	name = "gravity suspension harness"
+	desc = "A bootleg derivative of common Skrellian construction equipment, manufactured and heavily used by Deep Spacer tribes, this harness employs suspensor tech to either nullify or magnify gravity around the wearer."
+	slot_flags = ITEM_SLOT_BACK
+	icon_state = "gravityharness-off"
+	worn_icon_state = "gravityharness-off"
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/suit_digi.dmi'
+	worn_icon_teshari = 'modular_skyrat/master_files/icons/mob/clothing/species/teshari/back.dmi'
+	actions_types = list(/datum/action/item_action/toggle_mode)
+	w_class = WEIGHT_CLASS_HUGE
+	/// The current operating mode
+	var/mode = MODE_GRAVOFF
+	/// The cell that the harness is currently using
+	var/obj/item/stock_parts/cell/current_cell
+	/// If the cell cover is open or not
+	var/cell_cover_open = FALSE
+	/// If it's manipulating gravity at all.
+	var/gravity_on = FALSE
+	/// Defines sound to be played upon mode switching
+	var/modeswitch_sound = 'sound/effects/pop.ogg'
+	/// Max weight class of items in the storage.
+	var/max_w_class = WEIGHT_CLASS_NORMAL
+	/// Max combined weight of all items in the storage.
+	var/max_combined_w_class = 15
+	/// Max amount of items in the storage.
+	var/max_items = 7
+
+/obj/item/gravityharness/Initialize(mapload)
```

Comment:
Try to change the typepath into this everywhere, so it follows the `snake_case` conventions.
```suggestion
/obj/item/gravity_harness/Initialize(mapload)
```